src/sql/src/func.rs

```diff
@@ -1572,6 +1572,9 @@ lazy_static! {
             "replace" => Scalar {
                 params!(String, String, String) => VariadicFunc::Replace, 2087;
             },
+            "right" => Scalar {
+                params!(String, Int64) => BinaryFunc::Right, 42;
```

Comment:
This number is made up because I didn't know what the number should be at that time.

Reply:
42 was a good choice 😄